src/main/kotlin/org/jetbrains/research/transformations/Main.kt

```diff
@@ -1,5 +1,48 @@
 package org.jetbrains.research.transformations
 
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.ITree
+import com.github.gumtreediff.tree.TreeContext
+import java.io.File
+import org.jetbrains.research.transformations.Transformations.*
+
+fun setupParser() {
+    //copy parser file into /tmp directory, required by Gumtree
+    val sourcePath :String = "src/main/resources/pythonparser"
+    val targetPath :String = "/tmp/pythonparser"
+    val pythonparserFile :File = File(sourcePath)
+
+    try {
+        val targetFile :File = File(targetPath)
+        pythonparserFile.copyTo(targetFile)
+        val command = arrayOf("chmod", "+x", targetFile.getAbsolutePath())
+        val b = ProcessBuilder(*command)
```

Comment:
Please, don't use such short names for variables if they are **not** in the _for loops_ or in the _lambda expressions_. How about `builder`?